osgp/protocol-adapter-iec60870/osgp-iec60870/src/main/java/org/opensmartgridplatform/iec60870/Iec60870Server.java

```diff
@@ -42,4 +35,11 @@ public void stop() {
         LOGGER.info("Stopped IEC60870 Server.");
     }
 
+    public Iec60870ServerEventListener getIec60870ServerEventListener() {
+        return this.iec60870ServerEventListener;
+    }
+
+    public void pushAsdu(final ASdu asdu) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1186)